### ISSUE TITLE
Not showing the statuses of individual jobs in the Message

### ISSUE THREAD

**Issue by TalhaNaeem101** [Gamesight/slack-workflow-status#55]
Here is the job I wrote for slack notification for github action:
 ``` slack-workflow-status:
    if: always()
    name: Post Workflow Status To Slack
    needs: squash-merge
    runs-on: ubuntu-latest
    # actions.read permission is required.
    permissions:
      actions: 'read'
    steps:
      - name: Slack Workflow Notification
        uses: Gamesight/slack-workflow-status@v1.1.0
        with:
           # Required Input
          repo_token: ${{secrets.GITHUB_TOKEN}}
          slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
          include_jobs: 'true'
          # Optional Input
          channel: '#dps-cicd-flow'
          name: 'GitHub Action CI Bot'
          icon_emoji: ':squirrel:'
          icon_url: 'https://avatars0.githubusercontent.com/u/1701160?s=96&v=4'

```
And in the notification message it does not show (tick/cross) for succeeded and failed jobs. I am getting this notification in my slack channel:
![image](https://user-images.githubusercontent.com/105629434/211826172-48a0df2e-d2f2-4454-bf36-af85fbcf377c.png)

Ideally, it should show statuses for each individual jobs. 
Doc ref: https://github.com/marketplace/actions/slack-workflow-notifications
Ideally, it should show  ticks on succeeded jobs and cross on failed jobs like mentioned in their docs: 
![image](https://user-images.githubusercontent.com/105629434/211826792-d9cab3c3-6408-4e6d-9404-47fa1c081974.png)

Help is needed on P0, Please do help me achieve this using this workflow **ASAP**, Thanks

**Comment by rsyvarth** [comment]
Hi @TalhaNaeem101 - can you provide a screenshot of the workflow summary page from the github UI for this? From the look of the image you provided it seems that your job ran the `changes` job and the rest were cancelled (or possibly there was some issue fetching the status of those jobs).

**Comment by TalhaNaeem101** [comment]
This is the summary of GitHub action : 
![image](https://user-images.githubusercontent.com/105629434/211880960-a2e78cc2-861f-42dd-b836-f92ef7f5c7b9.png)

Basically, the job of the change detects the changes in directories and then builds the images for the services having the changes. And the case is that even if the job for a service (e.g. `superset-service`) ran but it was not shown on the slack message in the notification. And  one thing that is different in my case is that we are using reusable workflow for all jobs other than `changes` jobs.  @rsyvarth Please let me know if any other info/detail is required to understand my use-case.

**Comment by rsyvarth** [comment]
So seems like the request here is to hide skipped jobs - would that solve the issue you are seeing? You are expecting to only see jobs than ran, not those that are skipped in the status report?

**Comment by TalhaNaeem101** [comment]
yes ideally it should not show the skipped jobs, plus it should also show the jobs which were failed as cross with their names and the succeeded jobs as tick with their names.
PS: In my main workflow, every job is dependent on `changes` job. defined like this:
``` source-salesforce-build:
    name: source-salesforce
    needs: changes
    if: ${{ needs.changes.outputs.source-salesforce == 'true' }}
    uses: ./.github/workflows/reusable.yaml
    secrets: inherit
    with: 
      name: source-salesforce
      image_name: source-salesforce
      helm: false
      repo_type: ecr
```
Mainly wanted to highlight the usage of `needs` dependency among my jobs, I tried a dummy job without dependency of `changes` job and it worked fine, and showed the status correctly.  Please help me implement some solution for this, as I want to use this workflow in my use case. Would be highly thankful.

**Comment by TalhaNaeem101** [comment]
@rsyvarth

**Comment by rsyvarth** [comment]
Addressed in cddb573, released in v2.x. The new `hide_job_statuses` input takes a comma-separated list of job conclusions to omit from the rendered list — for your reusable-workflow case, `hide_job_statuses: 'skipped'` will hide the skipped entries while still showing successes and failures by name. Composes with `include_jobs: 'on-failure'` if you only want the section to appear on failure.
